packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -76,7 +77,15 @@ Future<void> main(List<String> args) async {
   final String xml = File(inputFilePath).readAsStringSync();
   final File outputFile =
       File(results['output'] as String? ?? '$inputFilePath.vg');
-  final Uint8List bytes = await encodeSvg(xml, args[0]);
+
+  bool maskingOptimizerEnabled = true;
+
+  if (results['optimize-masks'] == false) {
+    maskingOptimizerEnabled = false;
+  }
+
+  final Uint8List bytes = await encodeSvg(xml, args[0],
```

Comment:
nit: make all of these named parameters.

Reply:
fixed.